pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java

```diff
@@ -359,7 +359,8 @@ private void validateSchemaInternal(Schema schema) {
     validateSchemaName(schema.getSchemaName());
     try {
       List<TableConfig> tableConfigs = _pinotHelixResourceManager.getTableConfigsForSchema(schema.getSchemaName());
-      SchemaUtils.validate(schema, tableConfigs);
+      boolean isIgnoreCase = _pinotHelixResourceManager.isIgnoreCase();
+      SchemaUtils.validate(schema, tableConfigs, isIgnoreCase);
```

Comment:
i wonder if this is even necessary to extract this from helix resource manager. eventually this is a startup time config 
```
Boolean.parseBoolean(configs.getOrDefault(Helix.ENABLE_CASE_INSENSITIVE_KEY,
            Boolean.toString(Helix.DEFAULT_ENABLE_CASE_INSENSITIVE)));
```
we can simply encapsulate in the broker/controller starter?

Reply:
Thanks for you review @walterddr !
I think broker starter has already encapsulated this parameter.
```
boolean caseInsensitive =
        _brokerConf.getProperty(Helix.ENABLE_CASE_INSENSITIVE_KEY, Helix.DEFAULT_ENABLE_CASE_INSENSITIVE);
    TableCache tableCache = new TableCache(_propertyStore, caseInsensitive);
```
The reason why I extract this keyword from Helix ResouceManger is that PinotHelixResourceManger has extract this parameter from HelixAdmin and save it to Table Cache. So I don't think we need find another way to get this parameter. And PinotSchemaRestletResouce has injected an instance _pinotHelixResourceManager. So i don't that is a big deal.
```
Map<String, String> configs = _helixAdmin.getConfig(helixConfigScope,
        Arrays.asList(Helix.ENABLE_CASE_INSENSITIVE_KEY));
    boolean caseInsensitive = Boolean.parseBoolean(configs.getOrDefault(Helix.ENABLE_CASE_INSENSITIVE_KEY,
            Boolean.toString(Helix.DEFAULT_ENABLE_CASE_INSENSITIVE)));
    _tableCache = new TableCache(_propertyStore, caseInsensitive);
   ```

Reply:
no that's not what I meant. 
for example in `PinotQueryResource` it inject ControllerConf which has the case sensitivity config key in it presumably. instead of inject helix manager you can directly inject the config which is what we wanted here right?

Reply:
Hi @walterddr , I got that you mean and I have accepted your advice. Do you have any other advices?